classes/item.rb

```diff
@@ -12,6 +12,11 @@ def initialize(publish_date, archived)
     @genre = nil
   end
 
+  def add_label(label)
```

Comment:
Let's make the `add_label` method a setter using this syntax for a setter method:
  - `def add_label=(label)`
      `#your code here`
    `end`

When setting a value you would simply say: `item.add_label = item`

Be sure to do the same for [this method](https://github.com/AdornChoga/catalog-of-things/pull/27/files#diff-9cd9117ea5656fa3b21ec52b45dc18bc7b7a7d57a35b9d4168b152cb84231dedR13)  in label class which is required to be a setter method as well.

Reply:
👍